src/components/CtaSection.tsx

```diff
@@ -87,32 +105,39 @@ const Cta: FunctionComponent<Cta> = ({ text, icon, button = false, link, onClick
  * This is our CTA Section as defined in our DLS. Please refer to it for specs.
  *
  * @param props - component props
+ * @param props.background - background for the section
  * @param props.title - a title for the section
  * @param props.description - a description for the section
  * @param props.centerContent - a boolean to center the content
+ * @param props.slimWidth - a boolean for a slim width container
  * @param props.cta1 - cta item 1
  * @param props.cta2 - cta item 2
  */
 export const CtaSection: FunctionComponent<CtaSection> = ({
+    background,
     title = 'Get started with Sourcegraph',
     description = 'Understand, fix, and automate changes across your entire codebase.',
     centerContent = false,
+    slimWidth,
     cta1,
     cta2,
 }) => (
-    <ContentSection color="white" className="">
+    <ContentSection background={background}>
         <div
             className={classNames({
                 'tw-max-w-xl tw-mx-auto tw-text-center': centerContent,
                 'tw-grid tw-grid-cols-5': !centerContent,
```

Comment:
```suggestion
                'tw-grid tw-grid-cols-5': !centerContent && !slimWidth,
```

Reply:
Oh, thanks for catching this! I just realized how this was spanning since I had the `CtaSection` in the PostLayout. So the full width of the background was contained within the content container. I moved this outside of it to the blog slug so it spans across the full width of the page properly now.

Reply:
At the medium screen width I think it should start left aligning because it looks a bit off, no ?

<img width="816" alt="image" src="https://user-images.githubusercontent.com/59381432/187772288-33bb0700-7d3b-48d9-a8c0-cb46c11fe575.png">

Reply:
Yeah, I agree! With the grid approach, everything comes together at the centre so I added a modification to approach it with a combo of your max width suggestion and the grid with some padding on the description above tablet to get it as close as possible.

Reply:
It's beautiful! 😻  Thanks!